pkg/cmd/server/server.go

```diff
@@ -60,15 +63,36 @@ import (
 )
 
 func NewCommand() *cobra.Command {
-	var kubeconfig string
+	var (
+		kubeconfig   string
+		logLevelFlag = newLogLevelFlag()
+	)
 
 	var command = &cobra.Command{
 		Use:   "server",
 		Short: "Run the ark server",
 		Long:  "Run the ark server",
 		Run: func(c *cobra.Command, args []string) {
-			logger := logrus.New()
-			logger.Hooks.Add(&logging.ErrorLocationHook{})
+			var (
+				logLevel = logrus.InfoLevel
+				parsed   logrus.Level
```

Comment:
I wouldn't declare this; it's only used by the `ParseLevel` call, so I'd recommend
```go
if parsed, err := logrus.ParseLevel(logLevelFlag.String()); err == nil {
  logLevel = parsed
}
```